gammapy/estimators/utils.py

```diff
@@ -116,6 +118,80 @@ def find_peaks(image, threshold, min_distance=1):
     return table
 
 
+def find_peaks_in_flux_map(maps, threshold, min_distance=1):
+    """Find local TS peaks for a given Map.
+
+    Utilises the find_peaks function to find various parameters from FluxMaps.
+
+    Parameters
+    ----------
+    maps : `~gammapy.estimators.map.FluxMaps`
+    threshold : float or array-like
+        The TS data value or pixel-wise TS data values to be used for the
+        detection threshold.  A 2D ``threshold`` must have the same
+        shape as the map ``data``.
+    min_distance : int or `~astropy.units.Quantity`
+        Minimum distance between peaks. An integer value is interpreted
+        as pixels.
+
+    Returns
+    -------
+    output : `~astropy.table.Table`
+        Table with parameters of detected peaks
+    """
+    quantity_for_peaks = maps["sqrt_ts"]
+
+    if not isinstance(maps, FluxMaps):
+        raise TypeError("find_peaks_in_flux_map only supports FluxMaps")
+
+    if not quantity_for_peaks.geom.is_flat:
+        raise ValueError(
+            "find_peaks_in_flux_map only supports flat Maps, with no spatial axes of length 1."
+        )
+
+    table = find_peaks(quantity_for_peaks, threshold, min_distance)
+
+    if len(table) == 0:
+        return Table()
+
+    x = np.array(table["x"])
+    y = np.array(table["y"])
+
+    table.remove_column("value")
+
+    for name in maps.available_quantities:
+        values = maps[name].quantity
+        peaks = values[0, y, x]
+        table[name] = peaks
+
+    flux_data = maps["flux"].quantity
+    table["flux"] = flux_data[0, y, x]
+    flux_err_data = maps["flux_err"].quantity
+    table["flux_err"] = flux_err_data[0, y, x]
+
+    for column in table.colnames:
+        if column.startswith(("flux", "flux_err")):
```

Comment:
Is there a specific reason for defining a different format for the flux columns?

Reply:
For readability of the flux values -- typical fluxes of 1e-13, for example